docs/user_guide.rst

```diff
@@ -61,6 +62,50 @@ This sections describes storage backend operation supported by InfraSIM.
 * Drive erasure.
 
   Drive erasure feature is implemented in Qemu code. After erasing, all data residing on a disk drive will be overwritten with all zero. Below are examples of SAS and SATA drive erasure performed in Ubuntu 16.04.
+    
```

Comment:
I would suggest to move these usage on sg_format and hdparm into how-to section. Reason is it is describing how to exercise drive erasure feature, either infrasim implemented or physical driver native supported. 

Instead we could simple put a statement that sg-format and hdparm are supported to exercise our feature; if there're arguments supporting defining erasure duration, progressing, that content should be placed in user guide section.

Last comment is can we put a bullet as drive emulation (both feature, user guide chapters) and place drive erasure and INQUIRY/MODE SENSE data injection features under this bullet?

@JuneZhou19 @chenc52 thoughts?